src/awkward/_v2/cpp-headers/awkward/GrowableBuffer.h

```diff
@@ -278,6 +279,23 @@ namespace awkward {
       ptr_ = panel_.get();
     }
 
+    /// @brief Last element in last panel
+    PRIMITIVE
+    last() const {
+      if (ptr_->length() == 0) {
+        throw std::runtime_error("Buffer is empty");
```

Comment:
Eventually, these functions should be `noexcept` and have some default action for such cases. However, doing that after a performance-test framework is set up would be more useful, so that we can see how much it matters.